conf/struct.go

```diff
@@ -8,6 +8,9 @@ package conf
 // This is to ensure they both conform to the golang convention
 // and that they are able to be properly parsed by the `tok config-x` commands
 type Config struct {
+	Global struct {
+		Syncservice string `yaml:"syncservice"`
```

Comment:
This isn't a bool? What's the value expected for this field?

Reply:
It will be "docker" or "fusion"

Reply:
and maybe later we'll add others (eg: nfs)